sycl/doc/extensions/proposed/sycl_ext_oneapi_graph.asciidoc

```diff
@@ -872,17 +874,16 @@ to update an executable graph, every node in the graph must be either a kernel
 command or a host-task.
 
 The only characteristics that can differ between two topologically identical
-graphs during an update are:
+graphs during an update are the arguments to kernel nodes. For example,
+the graph may capture different values for the USM pointers or accessors used
+in the graph. It is these kernels arguments in `graph` that constitute the
+inputs & outputs to update to.
 
-* The arguments to kernel nodes may be different between graphs. For example,
-  the graph may capture different values for the USM pointers or accessors used
-  in the graph. It is these kernels arguments in `graph` that constitute the
-  inputs & outputs to update to.
+Differences in the following characteristics between two graphs during an
+update results in undefined behaviour:
 
-* The code that is executed in host-task nodes may be different between graphs,
-  and the update will reflect the code from `graph` host-tasks nodes. However,
-  any changes to executable code between kernel nodes are not counted as
-  topologically identical and is an exception with error code `invalid`.
+* Modifying the native C++ callable of a host_task node.
+* Modifying the {sycl-kernel-function}[kernel function] of a kernel node.
```

Comment:
Not seeing this link render nicely for some reason 
![image](https://github.com/reble/llvm/assets/2334151/a10244f6-4efc-4ef9-8d91-4ac88dbeb7c6)

Reply:
Ah apparently it doesn't like `sycl-kernel-function` being defined inside the table, moved it outside now!